packages/commonwealth/client/scripts/views/components/component_kit/new_designs/CWCommunityStakeBanner/CWCommunityStakeBanner.tsx

```diff
@@ -0,0 +1,54 @@
+import { useCommonNavigate } from 'navigation/helpers';
+import React from 'react';
+import CWBanner from 'views/components/component_kit/new_designs/CWBanner';
+import { CWText } from '../../cw_text';
+import { CWTag } from '../CWTag';
+
+type CWCommunityStakeBannerProps = {
+  onClose: () => void;
+  groupName: string;
+};
+export const CWCommunityStakeBanner = ({
```

Comment:
My opinion is that `CWComponents` should be design-system components that are building blocks for other components, eg `CWBanner`. CommunityStakeBanner is a specific implementation on top of CWBanner, thus, my proposition is that we should:
1. name the file & component `CommunityStakeBanner`
2. move it to `views/components/CommunityStakeBanner` directory

Reply:
Agreed and updated accordingly!